### PR TITLE
After Release fix

### PR DESCRIPTION
**Pull request by miroslavpojer**
Release Notes:
- Fixe typo in README.md.
- Added hint on how to get values.
- Removed fail-on-error from inputs.
- Removed valid from outputs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,12 +74,11 @@ See the default action step definition:
   env:
     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  
   with:
-    github-repository: "{ org }/{ repo }"
-    pr-numer: 109
+    github-repository: "{ org }/{ repo }"         # ${{ github.repository }}
```
Comment (MobiTikula) [inline README.md:77]: As I mentioned in VersionTagChecker project, this comment is little confusing. I see that the number below is the number of github.event. But for repository, I don't think it is true. It should be repository.login.owner and repository.login.name or something like it. OR it is same as repository.id.

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import pytest
+
+@pytest.fixture
+def mock_logging_setup(mocker):
+    """Fixture to mock the basic logging setup using pytest-mock."""
+    mock_log_config = mocker.patch("logging.basicConfig")
+    yield mock_log_config
+
+@pytest.fixture()
+def mock_exit(code) -> list:
```
Comment (MobiTikula) [inline tests/conftest.py:26]: Keep the same pattern of having or not having the method docstrings.

**Review by MobiTikula — COMMENTED** [review]
React to comments please.

File: README.md (hunk 1 above)
Comment (miroslavpojer) [reply README.md]: Fixed.

**Pull request merged**